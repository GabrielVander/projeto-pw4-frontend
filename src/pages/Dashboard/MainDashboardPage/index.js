import React from 'react';
import './styles.css';
import DashboardMenu from '../../../components/Dashboard/Menu';

function MainDashboardPage() {
	return (
		<>
			<DashboardMenu/>
		</>
	);
}

export default MainDashboardPage;