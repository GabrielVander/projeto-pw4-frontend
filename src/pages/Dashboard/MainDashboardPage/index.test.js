import React from 'react';
import { render } from '@testing-library/react';
import MainDashboardPage from './index';

describe('Component renders properly', () => {
	test('Matches snapshot', () => {
		const component = render(<MainDashboardPage/>);

		expect(component).toBeDefined();
		expect(component).toMatchSnapshot();
	});
});