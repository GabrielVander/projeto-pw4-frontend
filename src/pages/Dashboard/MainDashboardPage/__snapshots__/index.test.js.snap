// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders properly Matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header
        class="header"
      >
        <h2>
          Menu
        </h2>
      </header>
      <div
        class="card-deck"
      >
        <div>
          <div
            class="card bg-primary"
          >
            <div
              class="card-header"
            >
              Manage flows
            </div>
            <div
              class="card-body"
            >
              <svg
                fill="currentColor"
                height="50"
                size="50"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 16 16"
                width="50"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M6 1H1v14h5V1zm9 0h-5v5h5V1zm0 9h-5v5h5v-5zM0 1a1 1 0 011-1h5a1 1 0 011 1v14a1 1 0 01-1 1H1a1 1 0 01-1-1V1zm9 0a1 1 0 011-1h5a1 1 0 011 1v5a1 1 0 01-1 1h-5a1 1 0 01-1-1V1zm1 8a1 1 0 00-1 1v5a1 1 0 001 1h5a1 1 0 001-1v-5a1 1 0 00-1-1h-5z"
                  fill-rule="evenodd"
                />
              </svg>
            </div>
          </div>
        </div>
        <div>
          <div
            class="card bg-success"
          >
            <div
              class="card-header"
            >
              New document
            </div>
            <div
              class="card-body"
            >
              <svg
                fill="currentColor"
                height="50"
                size="50"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 16 16"
                width="50"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M9 1H4a2 2 0 00-2 2v10a2 2 0 002 2h5v-1H4a1 1 0 01-1-1V3a1 1 0 011-1h5v2.5A1.5 1.5 0 0010.5 6H13v2h1V6L9 1z"
                />
                <path
                  clip-rule="evenodd"
                  d="M13.5 10a.5.5 0 01.5.5v2a.5.5 0 01-.5.5h-2a.5.5 0 010-1H13v-1.5a.5.5 0 01.5-.5z"
                  fill-rule="evenodd"
                />
                <path
                  clip-rule="evenodd"
                  d="M13 12.5a.5.5 0 01.5-.5h2a.5.5 0 010 1H14v1.5a.5.5 0 01-1 0v-2z"
                  fill-rule="evenodd"
                />
              </svg>
            </div>
          </div>
        </div>
        <div>
          <div
            class="card bg-warning"
          >
            <div
              class="card-header"
            >
              View documents
            </div>
            <div
              class="card-body"
            >
              <svg
                fill="currentColor"
                height="50"
                size="50"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 16 16"
                width="50"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M3 2h8a2 2 0 012 2v10a2 2 0 01-2 2H3a2 2 0 01-2-2V4a2 2 0 012-2zm0 1a1 1 0 00-1 1v10a1 1 0 001 1h8a1 1 0 001-1V4a1 1 0 00-1-1H3z"
                  fill-rule="evenodd"
                />
                <path
                  d="M5 0h8a2 2 0 012 2v10a2 2 0 01-2 2v-1a1 1 0 001-1V2a1 1 0 00-1-1H5a1 1 0 00-1 1H3a2 2 0 012-2z"
                />
              </svg>
            </div>
          </div>
        </div>
        <div>
          <div
            class="card bg-danger"
          >
            <div
              class="card-header"
            >
              Manage workspaces
            </div>
            <div
              class="card-body"
            >
              <svg
                fill="currentColor"
                height="50"
                size="50"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 16 16"
                width="50"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M9.828 4a3 3 0 01-2.12-.879l-.83-.828A1 1 0 006.173 2H2.5a1 1 0 00-1 .981L1.546 4h-1L.5 3a2 2 0 012-2h3.672a2 2 0 011.414.586l.828.828A2 2 0 009.828 3v1z"
                />
                <path
                  clip-rule="evenodd"
                  d="M13.81 4H2.19a1 1 0 00-.996 1.09l.637 7a1 1 0 00.995.91h10.348a1 1 0 00.995-.91l.637-7A1 1 0 0013.81 4zM2.19 3A2 2 0 00.198 5.181l.637 7A2 2 0 002.826 14h10.348a2 2 0 001.991-1.819l.637-7A2 2 0 0013.81 3H2.19z"
                  fill-rule="evenodd"
                />
              </svg>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <header
      class="header"
    >
      <h2>
        Menu
      </h2>
    </header>
    <div
      class="card-deck"
    >
      <div>
        <div
          class="card bg-primary"
        >
          <div
            class="card-header"
          >
            Manage flows
          </div>
          <div
            class="card-body"
          >
            <svg
              fill="currentColor"
              height="50"
              size="50"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 16 16"
              width="50"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                clip-rule="evenodd"
                d="M6 1H1v14h5V1zm9 0h-5v5h5V1zm0 9h-5v5h5v-5zM0 1a1 1 0 011-1h5a1 1 0 011 1v14a1 1 0 01-1 1H1a1 1 0 01-1-1V1zm9 0a1 1 0 011-1h5a1 1 0 011 1v5a1 1 0 01-1 1h-5a1 1 0 01-1-1V1zm1 8a1 1 0 00-1 1v5a1 1 0 001 1h5a1 1 0 001-1v-5a1 1 0 00-1-1h-5z"
                fill-rule="evenodd"
              />
            </svg>
          </div>
        </div>
      </div>
      <div>
        <div
          class="card bg-success"
        >
          <div
            class="card-header"
          >
            New document
          </div>
          <div
            class="card-body"
          >
            <svg
              fill="currentColor"
              height="50"
              size="50"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 16 16"
              width="50"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M9 1H4a2 2 0 00-2 2v10a2 2 0 002 2h5v-1H4a1 1 0 01-1-1V3a1 1 0 011-1h5v2.5A1.5 1.5 0 0010.5 6H13v2h1V6L9 1z"
              />
              <path
                clip-rule="evenodd"
                d="M13.5 10a.5.5 0 01.5.5v2a.5.5 0 01-.5.5h-2a.5.5 0 010-1H13v-1.5a.5.5 0 01.5-.5z"
                fill-rule="evenodd"
              />
              <path
                clip-rule="evenodd"
                d="M13 12.5a.5.5 0 01.5-.5h2a.5.5 0 010 1H14v1.5a.5.5 0 01-1 0v-2z"
                fill-rule="evenodd"
              />
            </svg>
          </div>
        </div>
      </div>
      <div>
        <div
          class="card bg-warning"
        >
          <div
            class="card-header"
          >
            View documents
          </div>
          <div
            class="card-body"
          >
            <svg
              fill="currentColor"
              height="50"
              size="50"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 16 16"
              width="50"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                clip-rule="evenodd"
                d="M3 2h8a2 2 0 012 2v10a2 2 0 01-2 2H3a2 2 0 01-2-2V4a2 2 0 012-2zm0 1a1 1 0 00-1 1v10a1 1 0 001 1h8a1 1 0 001-1V4a1 1 0 00-1-1H3z"
                fill-rule="evenodd"
              />
              <path
                d="M5 0h8a2 2 0 012 2v10a2 2 0 01-2 2v-1a1 1 0 001-1V2a1 1 0 00-1-1H5a1 1 0 00-1 1H3a2 2 0 012-2z"
              />
            </svg>
          </div>
        </div>
      </div>
      <div>
        <div
          class="card bg-danger"
        >
          <div
            class="card-header"
          >
            Manage workspaces
          </div>
          <div
            class="card-body"
          >
            <svg
              fill="currentColor"
              height="50"
              size="50"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 16 16"
              width="50"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M9.828 4a3 3 0 01-2.12-.879l-.83-.828A1 1 0 006.173 2H2.5a1 1 0 00-1 .981L1.546 4h-1L.5 3a2 2 0 012-2h3.672a2 2 0 011.414.586l.828.828A2 2 0 009.828 3v1z"
              />
              <path
                clip-rule="evenodd"
                d="M13.81 4H2.19a1 1 0 00-.996 1.09l.637 7a1 1 0 00.995.91h10.348a1 1 0 00.995-.91l.637-7A1 1 0 0013.81 4zM2.19 3A2 2 0 00.198 5.181l.637 7A2 2 0 002.826 14h10.348a2 2 0 001.991-1.819l.637-7A2 2 0 0013.81 3H2.19z"
                fill-rule="evenodd"
              />
            </svg>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
